app/controllers/api/v1/rooms_controller.rb

```diff
@@ -39,13 +39,14 @@ def index
         render_json data: rooms + shared_rooms, status: :ok
       end
 
+      # GET /api/v1/rooms/:friendly_id.json
       def show
-        # TODO: samuel - should probably create/use a UUID instead of object id
         room = {
           id: @room.id,
           name: @room.name,
           presentation: room_presentation(@room),
-          created_at: @room.created_at.strftime('%A %B %e, %Y %l:%M%P')
+          created_at: @room.created_at.strftime('%A %B %e, %Y %l:%M%P'),
+          viewer_access_code: @room.viewer_access_code
```

Comment:
The front end shouldn't have access to the `access_code` because that would make it possible for the user to see it by inspecting the network tab. We should just return true or false if either of the access codes is present

Reply:
Hmmm that's a small problem, since this query is used in the room join as well

Reply:
2 potential solutions:
- Create a new endpoint for access codes
- Bundle along the access code with the RoomSettings query

Reply:
I will add a new endpoint to GET access codes. Thank you.